### ISSUE TITLE
TF Buffer: could not retrieve requested transform from buffer, unregistered. Drone not showing

### ISSUE THREAD

**Issue by AldoCF96** [PX4/avoidance#578]
**Describe the bug**
TF Buffer: could not retrieve requested transform from buffer, unregistered. Drone not showing.

**To Reproduce**
run simulation, close the simulation.

 run  roslaunch local_planner local_planner_stereo.launch but the drone never shows and it display message error as the tittle.

**Expected behavior**
exptect to show the  drone, and be able to connect to the drone

![98581176_234459078000553_6496001223311753216_n](https://user-images.githubusercontent.com/50414883/82022826-43afb380-965b-11ea-90e5-d7ac80e2d5c7.png)

**Comment by mrivi** [comment]
You probably haven't set up correctly the environment. Could you please share the outputs of 
```bash
echo $ROS_PACKAGE_PATH
echo $GAZEBO_MODEL_PATH
echo $GAZEBO_PLUGIN_PATH
echo $LD_LIBRARY_PATH

**Comment by AldoCF96** [comment]
![image](https://user-images.githubusercontent.com/50414883/82026136-511b6c80-9660-11ea-8527-9f8da46c578d.png)

Actually I never set up the plugin path it wasnt on the instructions

**Comment by mrivi** [comment]
You haven't run point 3 from https://github.com/PX4/avoidance#build-and-run-the-simulator correctly. Please run it again. The script `setup_gazebo.bash` sets those paths. Make sure to modify the paths to the Firmware to match your system

**Comment by AldoCF96** [comment]
I run it and I got this: 
aldo@aldo-TP501UB:~/Firmware$ echo $GAZEBO_PLUGIN_PATH
:/home/aldo/Firmware/build/px4_sitl_default/build_gazebo

It still shows the same error.  can you expand the part about modifying the paths ?

**Comment by AldoCF96** [comment]
I pulled the latest version of the code. is there something that I am missing ?

**Comment by AldoCF96** [comment]
FIxed: Apparently in Ubuntu 16.04 even when  it shows the PATH has been set up , you need to set up again, And run everything on the same terminal if you change terminal it wont work. 
 Need to test if it is better to sets all does paths with a screen and  run it from terminal seems too much work to  repeat these process all over again.
